plugins/generic/pln/classes/form/PLNStatusForm.inc.php

```diff
@@ -50,9 +50,9 @@ function display() {
 		
 		if (!$networkStatusMessage) {
 			if ($networkStatus === true) {
-				$networkStatusMessage = __(PLN_PLUGIN_NOTIFICATION_PLN_ACCEPTING);
+				$networkStatusMessage = __("plugins.generic.pln.notifications.pln_accepting");
```

Comment:
Sorry, I'm getting trivial here -- but quoted strings that don't include variable replacements should use ' rather than " to delimit.